README.md

```diff
@@ -0,0 +1,2 @@
+# Hello Microverse Project
+ 
```

Comment:
- [ ] It's important to create a more professional README file this is important as it provides details about your project.
You can use this [template](https://github.com/microverseinc/readme-template/blob/master/README.md)  to create your README file.